### PR TITLE
add standard stream projections

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

File: src/Container/AllStreamProjectionFactory.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+     * @var array
+     */
+    private $pdoDriverSchemeAliases = [
+        'pdo_mysql'  => 'mysql',
+        'pdo_pgsql'  => 'pgsql',
+    ];
+
+    private $pdoDriverSchemeSeparators = [
+        'pdo_mysql'  => ';',
+        'pdo_pgsql'  => ' ',
```
Comment (lucascourot) [inline src/Container/AllStreamProjectionFactory.php]: double space between `'pdo_mysql'`  and `=>`. Same above.

**Review by lucascourot — APPROVED** [review]

File: src/Container/AllStreamProjectionFactory.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                    $this->getPDOConnection($container, $config),
+                    '$all',
+                    $config['event_streams_table'],
+                    $config['projections_table'],
+                    $config['lock_timeout_ms'],
+                    true
+                );
+                break;
+            case 'postgres':
+                $projection = new \Prooph\EventStore\PDO\Projection\PostgresEventStoreProjection(
```
Comment (basz) [inline src/Container/AllStreamProjectionFactory.php]: would need a suggestion or a requirement in composer.json

File: src/Container/AllStreamProjectionFactory.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    public function __invoke(ContainerInterface $container): AllStreamProjection
+    {
+        $config = $container->get('config');
+        $config = $this->options($config);
+
+        $eventStore = $container->get($config['event_store']);
+
+        switch (strtolower($config['type'])) {
+            case 'mysql':
+                $projection = new \Prooph\EventStore\PDO\Projection\MySQLEventStoreProjection(
```
Comment (basz) [inline src/Container/AllStreamProjectionFactory.php]: would need a suggestion or a requirement in composer.json

**Review by basz — APPROVED** [review]

File: src/Container/AllStreamProjectionFactory.php (hunk 3 above)
Comment (prolic) [reply src/Container/AllStreamProjectionFactory.php]: done

**Pull request merged**